finalPrep/modZeroHero.js

```diff
@@ -1,40 +1,79 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "Jean Gray/Phoenix"
+var specialAbility = "telepathy"
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "Well howdy" + heroName + "!"
+var catchphrase = `I am the queen of ${specialAbility}!`
+
+
 // Declare two variables - power AND energy - set to integers
+var power = 8
+var energy = 10
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
+var fullPower = power * 500
+var fullEnergy = energy + 150
 
-// Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+// Declare two variables - isHuman and identityConcealed - assigned to booleans
+var isHuman = true
+var identityConcealed = true
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ["Thanos", "The Joker", "Lex Luthor"]
+var sidekicks = ["Robin", "Chewbacca", "Samwise Gamgee"]
 
 // Print the first sidekick to your console
+console.log(sidekicks[0])
 
 // Print the last archEnemy to the console
+console.log(archEnemies[2])
 
 // Write some code to add a new archEnemy to the archEnemies array
+var newLength = archEnemies.push("James Moriarty")
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies)
 
 // Remove the first sidekick from the sidekicks array
+var newLength = sidekicks.shift()
 
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks)
+//**Above you asked us to REMOVE the first sidekick and here you're asking us to log to ensure that we ADDED a sidekick. I'm going to assume you meant to log to make sure I removed it.**
+
 
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+function assessSituation(dangerLevel, saveTheDay, badExcuse){
+  if (dangerLevel > 50){
+    console.log(badExcuse);
+  } else if (dangerLevel >= 10 && dangerLevel <=50) {
```

Comment:
A tip here- when setting up conditions, you can see if it works to make the most complex logic live in your `else` block, so you can skip typing it out.